modules/custom/d_commerce_product/config/install/commerce_payment.commerce_payment_gateway.payment_gateway_test.yml

```diff
@@ -0,0 +1,19 @@
+uuid: 5f87c636-8fcb-4758-afef-ccfbc3974cb5
```

Comment:
Eksportowałeś to z features? Bo UUID rozsypują strony oparte na dystrybucji, features to wycina zazwyczaj

Reply:
Ok, ręcznie sobie brałem pare plików. Poprawie za chwile.